build-support/src/main/kotlin/platforms.kt

```diff
@@ -46,7 +48,7 @@ val linuxTargets = listOf(
 
 val nativeTargets = appleTargets + linuxTargets + mingwTargets
 
-/** Note that size_t is 32-bit on all watchOS versions (ie. pointers are always 32-bit). */
+/** Note that size_t is 32-bit on legacy watchOS versions (ie. pointers are always 32-bit). */
```

Comment:
Why the change to the word legacy? Arm64 is the architecture of the current Apple watch.

Reply:
Woops sorry, you're right, I think I got confused because `watchosX64` is 64 bits and specific to recent WatchOS, but that's only the simulator, not the actual watch.